can-define.js

```diff
@@ -288,6 +288,7 @@ make = {
 		events: function(prop, getCurrent, setData, eventType) {
 			return function(newVal) {
 				if (this.__inSetup) {
+					Observation.ignore(getCurrent).call(this);
```

Comment:
Honestly don't know if I do.  I was trying to avoid regressing on the observable value problem.  Let's loop in @christopherjbaker .